### PR TITLE
Replaced FreeGPT35 with aurora

### PR DESCRIPTION
**Pull request by chris-ss**

### CODE REVIEW

**Comment by chris-ss** [comment]
Replaced FreeGPT35 with aurora

**Review by ilkocn — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -36,29 +36,39 @@ This project is still in its very early days. Expect some bugs.
 ### Run the latest release
 
 ```bash
-git clone https://github.com/nashsu/FreeAskInternet.git
+git clone https://github.com/chris-ss/FreeAskInternet.git
```
Comment (amiantos) [inline README.md:39]: This PR is changing the readme URLs of the repo to the fork. FYI @ilkocn 

Also changes the docker container away from nashsu account to duriancat.

Seems like the changes are unrelated to the PR so just making sure...

Comment (Mandeep0001) [reply README.md]: Correct
